### ISSUE TITLE
Execution Block ID uid://A001/X15a0/Xb8 Sgr_A_st_e_03_TM1

### ISSUE THREAD

**Issue by keflavich** [ACES-CMZ/reduction_ACES#147]
Sgr_A_st_e_03_TM1
[uid://A001/X15a0/Xb8](https://almascience.org/aq/?result_view=observation&mous=uid://A001/X15a0/Xb8)

* [x] Observations completed?
* [ ] Delivered?
* [ ] Downloaded? (specify where)
  * [ ] hipergator
  * [ ] hipergator pipeline run

* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_e_03_TM1/html/) unpacked
* [ ] [Weblog](https://data.rc.ufl.edu/secure/adamginsburg/ACES/weblogs/humanreadable/Sgr_A_st_e_03_TM1/html/) Quality Assessment?
* [ ] Imaging: Continuum
* [ ] Imaging: Lines

  * [ ] SPW25 H13CN
  * [ ] SPW27 H13CO+/SiO
  * [ ] SPW29 HCO+
  * [ ] SPW31 HNCO
  * [ ] SPW33 Cont 1
  * [ ] SPW35 Cont 2

**Comment by keflavich** [comment]
Ah... rather, it does, but the EB was only partially completed:

![image](https://user-images.githubusercontent.com/143715/179985406-0c76fb8c-59f6-4b35-b997-f69518de0ccd.png)

The link shows that little triangle in the bottom-left.

**Comment by d-l-walker** [comment]
Overall this looks good. Once SPW 33 has been imaged, this should be good-to-go.
Note: this is another instance where the beam size is right at the upper edge of our range, but only because `robust=0` was used.

**Comment by keflavich** [comment]
spw33+35 continuum need reimaging

**Comment by d-l-walker** [comment]
@keflavich is this related to the `datacolumn` issue that you've been looking into?

I'm currently running the re-cleaning for SPW 33 with a higher `cyclefactor` due to it diverging. Do you need me to do 35 too? Or have you triggered that on your end already?

**Comment by keflavich** [comment]
yeah it reimaged successfully no problem.  [This](https://github.com/ACES-CMZ/reduction_ACES/issues/147#issuecomment-1474391788) issue was only continuum.

**Comment by d-l-walker** [comment]
SPW 33 re-imaging attempt still diverged even with `cyclefactor = 2.0` (max intensity map below). Emission looks neat though. Going to bump up to 2.5 and hope ...

![Screenshot 2023-03-21 at 17 36 22](https://user-images.githubusercontent.com/13015266/226694953-b8a65012-fa4b-4b6e-8e4d-7ccd558cdc11.png)

**Comment by d-l-walker** [comment]
SPW 33 re-imaging still diverged with `cyclefactor = 2.5`. Trying with `3.0` ...

**Comment by d-l-walker** [comment]
SPW 33 was successfully imaged using `cyclefactor=3.0`. 
Products have been added to `/upload/Sgr_A_st_e_03_TM1_reclean/` @keflavich.
I'll update the tclean commands file now.

<img width="474" alt="Screenshot 2023-04-26 at 18 02 23" src="https://user-images.githubusercontent.com/13015266/235245866-e2461927-4489-4da6-9403-cca5b345e2e6.png">

**Comment by keflavich** [comment]
I moved the recleaned files to `/orange/adamginsburg/ACES/data/2021.1.00172.L/science_goal.uid___A001_X1590_X30a8/group.uid___A001_X1590_X30a9/member.uid___A001_X15a0_Xb8/reclean`, then I symlinked them into the current filenames with:
`for fn in ../../reclean/*; do ln -s $fn $(basename ${fn/.reclean/}); done`

**Comment by d-l-walker** [comment]
QA for latest continuum selection/imaging:
* New images look nice, especially SPW 33 + 35!
* The structure in 25 + 27 looks mostly similar, but the diffuse emission looks more prominent. Tricky to tell because of the higher noise. Nothing obvious in the MFS images. I'll download the cubes to check.

![region_e_continuum](https://github.com/ACES-CMZ/reduction_ACES/assets/13015266/73fba691-4bf2-4bed-b045-700c305ede2c)

**Comment by d-l-walker** [comment]
Still quite a lot of large scale emission in the low freq image, but less than before. Structure now looks more similar to that in the higher freq image.

Left: old images
Right: new images
![Screenshot 2024-03-27 at 12 22 27](https://github.com/ACES-CMZ/reduction_ACES/assets/13015266/78db1ae5-9b97-4b94-a037-2206b9958837)

**Comment by keflavich** [comment]
There are clearly a few channels with line emission, unfortunately:

https://data.rc.ufl.edu/secure/adamginsburg/ACES/diagnostic_plots/spectra/uid___A001_X15a0_Xb8.s38_0.Sgr_A_star_sci.spw35.cube.I.iter1.image_diagnostic_spectra.png
![image](https://github.com/ACES-CMZ/reduction_ACES/assets/143715/82096072-6dd3-4fdd-a1b6-3ef0ca4d204f)
